### PR TITLE
add debug to the indexed artifacts

### PR DESCRIPTION
**Pull request by tarekziade**

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
bors try

**Comment by pocmo** [comment]
bors try

File: taskcluster/ci/signing/kind.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ job-template:
           default: autograph_apk
     index:
         by-build-type:
-            (fennec-.+|nightly|performance-test|beta|production):
+            (fennec-.+|nightly|performance-test|beta|production|debug):
```
Comment (pocmo) [inline taskcluster/ci/signing/kind.yml:65]: What's the reason for that? They won't be signed so not really installable without manual work.

**Comment by pocmo** [comment]
Ah, Fenix repo, bors was removed. 😞

**Comment by JohanLorenzo** [comment]
Sadly, we don't want bors on Fenix anymore. See https://github.com/mozilla-mobile/fenix/issues/9295#issuecomment-612073112 for more context.

**Comment by tarekziade** [comment]
Thanks @JohanLorenzo  @pocmo

**Comment by JohanLorenzo** [comment]
> I'm fine landing this patch as is. I'm almost sure it won't break master and I'll have a close look at Treeherder to be 100% positive.

I'm glad I didn't say I was 100% sure 😅. This PR requires https://phabricator.services.mozilla.com/D70809. I'm on it.

**Pull request merged**